frontend/src/components/intake/ShasIntakeSchema.ts

```diff
@@ -0,0 +1,144 @@
+import { array, boolean, mixed, number, object, string } from 'yup';
+
+import {
+  ContactPreferenceList,
+  NumResidentialUnits,
+  ProjectRelationshipList,
+  YesNo,
+  YesNoUnsure
+} from '@/utils/constants';
+import { BASIC_RESPONSES } from '@/utils/enums';
+
+// Form validation schema
+const YesNoUnsureSchema = string().required().oneOf(YesNoUnsure);
+const prevYesSelected = (prevQuestion: string) => prevQuestion === BASIC_RESPONSES.YES;
+const stringRequired = string().required().max(255);
+
+export const getIntakeSchema = (ProjectLocation: Array<string>) =>
+  object({
+    applicant: object({
+      firstName: stringRequired.label('First name'),
+      lastName: stringRequired.label('Last name'),
+      phoneNumber: stringRequired.label('Phone number'),
+      email: stringRequired.label('Email'),
+      relationshipToProject: string().required().oneOf(ProjectRelationshipList).label('Relationship to project'),
+      contactPreference: string().required().oneOf(ContactPreferenceList).label('Contact Preference')
+    }),
+    basic: object({
+      isDevelopedByCompanyOrOrg: string().required().oneOf(YesNo).label('Project developed'),
+      isDevelopedInBC: string().when('isDevelopedByCompanyOrOrg', {
+        is: (previousQuestion: string) => previousQuestion === BASIC_RESPONSES.YES,
```

Comment:
This naming threw me off for a bit. May be better to just stick to calling it `value` as there is no true context tie to the previous form value outside of the current layout. The actual value coming in is whatever is defined in the `when`.